addons/base_vat/models/res_partner.py

```diff
@@ -252,20 +260,25 @@ def check_vat_al(self, vat):
 
     __check_tin_hu_individual_re = re.compile(r'^8\d{9}$')
     __check_tin_hu_companies_re = re.compile(r'^\d{8}-[1-5]-\d{2}$')
```

Comment:
Don't we need to make sure it doesn't start with an `8` ?

Reply:
I couldn't find any reference about that, so I guess no...